source/client/scripts/stars.js

```diff
@@ -1,78 +1,122 @@
-let mouseX = 0;
-let mouseY = 0;
-let stars = [];
-
-const starSize = 3;
-const starSlowness = 700;
-const maxOpacity = 0.8;
-const mouseSpeed = 0.2; // how much does the mouse affect the stars?
-
-const starsContainer = document.getElementById('stars-container');
-
-window.addEventListener('DOMContentLoaded', () => {
-  stars = new Array(400).fill().map(() => ({
-    x: Math.random(),
-    y: Math.random(),
-    z: Math.random(),
-  }));
-
-  for (let i = 0; i < stars.length; i++) {
-    const starElement = document.createElement('div');
-    starElement.className = 'star';
-    updateStar(starElement, i);
-    starsContainer.appendChild(starElement);
-  }
-
-  animate();
-});
+/**
+ * @typedef { {
+ *  xPosPercent: number,
+ *  yPosPercent: number,
+ *  zPosPercent: number,
+ *  htmlElement: HTMLElement
+ * } } Star
+ */
+
+const NUM_STARS = 200;
+const MAX_OPACITY_PERCENT = 0.65;
+const STAR_SPEED_PERCENT = 0.0025;
```

Comment:
Might need to decrease speed, at the moment the stars are quite a bit faster than before, making the mouse movement and parallax effects weaker

Reply:
Good point; do you want to try some different values for star speed and camera effect?

Reply:
I threw some new numbers in there if you want to check them out. I also excluded a centered circle from the possible locations, so that they don't conflict with the logo and buttons like OG implementation.

Reply:
I think if anything the stars should be on the faster side since there's so few stars now, so making them fast will make them more noticeable.